specification/validate_includes.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+import argparse
+from os import listdir
+import sys
+import re
+from os.path import isfile, join
+
+def compile_includes(keyname):
```

Comment:
nit: we're no longer compiling these. Not a huge deal if we don't update.

Reply:
Added push to fix this. Good catch